### ISSUE TITLE
Refactor current-projects file to fix any formatting issue with new tools data format

### ISSUE THREAD

**Issue by ronaldpaek** [hackforla/website#4860]
### Overview

The purpose of this issue is to refactor the code in the current-project file to handle a list of strings(tools), instead of a single one. This change will maintain consistent formatting and resolve problems introduced by the recent change in data format.

### Action Items

- [x] Refer to [current-projects.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/current-projects.js) and locate where tools is being referenced
- [x] Refactor this code so tools can work as a comma separated String or an Array
```
{%- if project.tools -%},
"tools": `{{ project.tools }}`
{%- endif -%}
```
to:
```
{%- if project.tools -%},
"tools": {{ project.tools | jsonify }}
{%- endif -%}
```

- [x] Refactor this code to map over tools since tools will be an Array now and not a String

```
${project.tools ?
`
<div class="project-tools">
<strong>Tools: </strong>
${ project.tools }
</div>
`:""
}
```
to:
```
${project.tools ?
`
<div class="project-tools">
<strong>Tools: </strong>
${(Array.isArray(project.tools) ? project.tools : project.tools.split(','))
    .map(tool => `<p class='project-card-field-inline'> ${tool}</p>`)
    .join(", ")
}
</div>
`: ""
}
```
After completing both changes this should allow us to make changes to tools without effecting the functionality of the website
- [x] Confirm that the Tools data is rendering correctly/ as before on the Projects page.

Merge team:
- [ ] Once this issue is merged, release its dependency on #4777

### Resources/Instructions

- [Projects](https://www.hackforla.org/projects/) page
- [Liquid template language](https://shopify.github.io/liquid/)
-  This issue is part of an epic: #4427

**Comment by ExperimentsInHonesty** [comment]
Hi @ronaldpaek, what is the state of each of the images you have provided.  Is one the current state and the second the future state?  And if so, the second one is not showing spaces between, and that doesn't look good.  It was my understanding that this change was just to made it possible to do the projects check page easier using the standards of the technologies field.  Not sure why there would be any visual change.

**Comment by ronaldpaek** [comment]
@ExperimentsInHonesty, I see, yes those images were of the two state of the data in as one long string with commas or in a list of strings. I wanted to verify that the code would not break and cause the an error.

I think we should change all the data to list format first then, but once I think either way there will be some minor spacing issues with the commas till all the revisions are done, but the code shouldn't break though.

**Comment by t-will-gillis** [comment]
@ExperimentsInHonesty I confirmed that @ronaldpaek's changes will render the Tools data correctly before, during, and after the transition from the string to the list/array datatype. I have deleted the image since it is no longer relevant to the issue, and I will hide the comments.

**Comment by ronaldpaek** [comment]
Did you end up doing this?
